README.md

```diff
@@ -75,7 +82,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var newTemp = [];
+var pizza = {temperature:70, method:"bake"};
+setTemp (pizza);
+function setTemp (newTemp) {
+newTemp.temperature = 300;
```

Comment:
remember array's don't have properties! What if you wrote your function `setTemp` as method on `pizza` to update the values of your `pizza`'s `temperature`, i.e. :

```js
var pizza = {
  temperature: 70,
  bake: function(){
    this.temperature = 300;
  }
}
```